README.md

```diff
@@ -25,7 +25,7 @@ Chrome Platform Status
 5. Install pip for python2
```

Comment:
Perhaps we can remove this part now and `npm run dev-deps`

Reply:
We probably need to rewrite README.md assuming that the developer only has py3.  Then, we could write a separate README-py2.md for those rare times when someone needs to do py2 development.  The new py2 instructions will look a lot different because it will basically say to test things on staging rather than do much locally.